pom.xml

```diff
@@ -47,7 +47,7 @@
 
    <properties>
       <feign.extension.version>9.3.1</feign.extension.version>
-      <hawkbit.version>0.2.0M7</hawkbit.version>
+      <hawkbit.version>0.2.0-SNAPSHOT</hawkbit.version>
```

Comment:
Your change is going into 0.3.0-SNAPSHOT line. May I ask you to update it accordingly?